front/src/actions/house.js

```diff
@@ -64,6 +64,31 @@ function createActions(store) {
       });
       store.setState(newState);
     },
+    updateHouseAlarmCode(state, code, houseIndex) {
+      const newState = update(state, {
+        houses: {
+          [houseIndex]: {
+            alarm_code: {
+              $set: code
+            }
+          }
+        }
+      });
+      store.setState(newState);
+    },
+    updateHouseDelayBeforeArming(state, delayBeforeArming, houseIndex) {
+      console.log(delayBeforeArming);
```

Comment:
I think you need to remove `console.log`

Reply:
done